packages/flutter/test/widgets/text_test.dart

```diff
@@ -47,7 +47,7 @@ void main() {
     expect(text, isNotNull);
     expect(text.textScaleFactor, 3.0);
     final Size largeSize = tester.getSize(find.byType(RichText));
-    expect(largeSize.width, equals(211.0));
+    expect(largeSize.width, equals(210.0));
```

Comment:
This should be 210 exactly (5 characters * 14 pixels font size * 3 scale factor = 210).
In the tests we use a font that has ASCII characters that are exactly the font size high and exactly the font size wide, (with their baseline at exactly 2/10th of the way from the bottom).